ingest/src/main/java/com/microsoft/azure/kusto/ingest/StreamingIngestClient.java

```diff
@@ -154,7 +154,7 @@ public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, Inges
     private String getFormat(IngestionProperties ingestionProperties) {
         String format = ingestionProperties.getDataFormat();
         if (format == null) {
-            return "csv";
+            return "Csv";
```

Comment:
Why uppercase?

Reply:
Unfortunately these are case sensitive ATM

Reply:
decided to leave it lower case - this will work on next kusto versions

Reply:
This here acutally is just a mistake of the intelij refactoring